packages/core/src/tools/display.ts

```diff
@@ -7,7 +7,15 @@
  * but we should be safe from infinite loop nonetheless.
  */
 
-import { ConsoleApiName } from '../domain/console/consoleObservable'
+export const ConsoleApiName = {
```

Comment:
I'd find more logic to have it in `domain/console/consoleApiNames.ts`
Wdyt?

Reply:
Hum, okay then we can keep it here. I'm not sure having it in `browser/consoleApiNames` will make things clearer. 

Note for later: If we really don't want to have domain dependencies out of `/domain` it could be nice to add an eslint rule.